{{cookiecutter.project_slug}}/tools/run_creator.py

```diff
@@ -39,8 +39,8 @@ def get_output_config(folder: Path) -> Dict:
     for input_key, input_value in input_config["inputs"].items():
```

Comment:
I would always have a minimal ``if __name__=="__main__"`` section and create a ``main`` function. IMHO it is as a very good practice that makes that part of the code testable, reusable and composable (see other comments)

Reply:
you're right. Done and added also sys.exit etc.